torch_xla/__init__.py

```diff
@@ -32,17 +32,7 @@ def _setup_xla_flags():
   os.environ['XLA_FLAGS'] = ' '.join(flags)
 
 
-def _set_missing_env(name, value):
-  if name not in os.environ:
-    os.environ[name] = value
-
-
-def _setup_default_env():
-  _set_missing_env('TF_CPP_MIN_LOG_LEVEL', '1')
-
-
 # These needs to be called before the _XLAC module is loaded.
-_setup_default_env()
```

Comment:
If this is not set, what's the return value of the `MinLogLevelFromEnv`, will be be 0?

Reply:
Yes, https://github.com/tensorflow/tensorflow/blob/efff893c709b585da9771cb3a2997321efef36ae/tensorflow/core/platform/default/logging.cc#L70